THIRD-PARTY-NOTICES.txt

```diff
@@ -37,4 +37,21 @@ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
 
 http://www.apache.org/licenses/LICENSE-2.0
 
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
\ No newline at end of file
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+
+License notice for StyleCop Analyzers
```

Comment:
Is all the light-up related code from StyleCop analyzers?

Reply:
Yes